diff --git a/lib/es/stylex.d.ts b/lib/es/stylex.d.ts
index 4e8184120bd50f6b5ae03a34a319519e11c4db2c..732646e6f6a4b58915741af6436e5ac9f806ac6b 100644
--- a/lib/es/stylex.d.ts
+++ b/lib/es/stylex.d.ts
@@ -247,3 +247,6 @@ type IStyleX = {
 };
 export declare const legacyMerge: IStyleX;
 export declare type legacyMerge = typeof legacyMerge;
+
+import * as stylex from './stylex';
+export { stylex as default };
diff --git a/lib/es/stylex.mjs b/lib/es/stylex.mjs
index 5a8fe56a45590b80d2334a3d88fe12bbedb0e7c1..a74dd4578eac7f406686d7cc0b41e24cef14e2b7 100644
--- a/lib/es/stylex.mjs
+++ b/lib/es/stylex.mjs
@@ -294,3 +294,6 @@ const legacyMerge = /*@__PURE__*/ function () {
 }();
 
 export { attrs, create, createTheme, defaultMarker, defineConsts, defineMarker, defineVars, env, firstThatWorks, keyframes, legacyMerge, positionTry, props, types, unstable_conditional, unstable_createThemeNested, unstable_defineConstsNested, unstable_defineVarsNested, viewTransitionClass, when };
+
+import * as stylex from './stylex.mjs';
+export { stylex as default };
